### DISCUSSION TITLE
Have any plan to support more storage service?

### DISCUSSION THREAD

**Discussion by alex-guoba** [caching-tools/next-shared-cache#484] [Ideas]
Hi, I just wanna know did we have any plan to support more handlers like postgres or Mysql ?

**Comment by better-salmon** [comment]
@alex-guoba hello! Thank you for your question! `@neshca/cache-handler` can utilize any cache storage, it is just a matter of choice. However, databases are usually slow, so, we have created Handlers for the fastest type of store. Next prebuilt Handler probably will be a Handler for S3 or a utility Handler for dealing with the sitemap.

You can create your own Postgres or Mysql Handler and share with the community.

**Reply by alex-guoba** [reply]
OK, thanks for your reply. I will try it later.
